src/main/java/bitflow4j/Header.java

```diff
@@ -16,26 +18,19 @@ public Header(String[] header) {
         this.header = header;
     }
 
-    public boolean hasChanged(Header oldHeader) {
```

Comment:
Leave this method as a proxy to call equals. Will save some refactoring in repos that build uppon this.